spec/requests/volunteers_spec.rb

```diff
@@ -154,13 +154,14 @@
     context "with valid params" do
       it "updates the volunteer" do
         patch volunteer_path(volunteer), params: {
-          volunteer: {email: "newemail@gmail.com", display_name: "New Name"}
+          volunteer: {email: "newemail@gmail.com", display_name: "New Name", phone_number: "+14163218092"}
```

Comment:
This looks too much like a real phone number, please replace it with an obviously fake phone number

Reply:
fixed